### ISSUE TITLE
OpenHistoricalMap

### ISSUE THREAD

**Issue by 1ec5** [wikimaps-dev/mapwarper#126]
Wikimaps Warper integrates with OpenStreetMap accounts, OSM Carto, and an instance of iD configured for OSM. Historical maps are [much more relevant to OpenHistoricalMap](https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/Tracing_Old_Maps_and_Imagery), so there should be parallel integration with OHM wherever possible.

OHM mostly runs on forks of OSM software, which should make this mostly a matter of duplicating the OSM integration code. One catch is that OHM doesn’t publish a raster tileset to use as a basemap. If an OHM basemap is desired, OpenLayers does have some support for MVT and possibly MapLibre styles.

**Comment by timwaters** [comment]
Many thanks for the issue,

in the trace tab of a map, there is a link to OpenHistoricalMap which opens
the editor at the correct area and adds the rectified image as a basemap.
e.g. https://warper.wmflabs.org/maps/7835#Trace_tab
It also adds the source and a comment to the changeset

There used to be a separate embedded id editor within this trace tab, but
now that OHM is running allowing basemaps to be passed through, this was
removed. (I was a maintainer of both OHM and the warper at that time). OHM
and Wikimedia Commons Warper has grown together

There could be the question of adding the trace link to the wider mapwarper
project, and a question of integrating OHM tiles into wikimaps warper too

**Comment by 1ec5** [comment]
Yep, that’s wonderful, thanks! I wonder if it would be useful to allow the user to log in using an OHM account, similar to the OSM option. Personally, I only ever log in via Commons, since I want to update the infobox on the image description page, but there may be an occasional OHM user who doesn’t have a Wikimedia account.

**Comment by timwaters** [comment]
Good idea! https://github.com/wikimaps-dev/mapwarper/issues/127
